### PR TITLE
update mappings

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: snippet-extractor-output/snippets.yaml [hunk 1]
```diff
@@ -316,7 +316,7 @@ Excel.CustomXmlPartCollection.set:
         targetRange.format.autofitColumns();
         await context.sync();
     });
-Excel.CustomXmlPart.add:
+Excel.CustomXmlPartCollection.add:
```
Comment (AlexJerabek) [inline snippet-extractor-output/snippets.yaml]: Should this target CustomXMLPart.getXml instead?

File: snippet-extractor-output/snippets.yaml [hunk 2]
```diff
@@ -482,7 +482,7 @@ Excel.CustomXmlPartScopedCollection.getItemOrNullObject:
 
         await context.sync();
     });
-Excel.CustomXmlPartScopedCollection.getByNamespace:
```
Comment (AlexJerabek) [inline snippet-extractor-output/snippets.yaml]: Maybe this should be a general example of Excel.CustomXmlPartScopedCollection (not an individual method).

File: snippet-extractor-output/snippets.yaml [hunk 3]
```diff
@@ -512,7 +512,7 @@ Excel.CustomXmlPartScopedCollection.getByNamespace:
 
         await context.sync();
     });
-Excel.Worksheet.getCount:
+Excel.CustomXmlPartCollection.getCount:
```
Comment (AlexJerabek) [inline snippet-extractor-output/snippets.yaml]: Should this be CustomXmlPartScopedCollection?

File: snippet-extractor-output/snippets.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply snippet-extractor-output/snippets.yaml]: The same snippet is already/also tagged for CustomXMLPart.getXml

File: snippet-extractor-output/snippets.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply snippet-extractor-output/snippets.yaml]: the property getByNamespace is of type Excel.CustomXmlPartScopedCollection but belongs to Excel.CustomXmlPartCollection which is why I've set it like that. If anything, there could be an argument for setting the example at Excel.CustomXmlPartCollection level.

Comment (AlexJerabek) [reply snippet-extractor-output/snippets.yaml]: Right. I guess my thought was this snippet is "some stuff done to a scoped collection", so it should be there. I don't have a strong opinion, so I trust your judgement.

Comment (ElizabethSamuel-MSFT) [reply snippet-extractor-output/snippets.yaml]: Actually, I should say it returns a Scoped one.

**Pull request merged**